allennlp/tests/training/trainer_test.py

```diff
@@ -121,6 +122,26 @@ def test_passing_trainer_multiple_gpus_raises_error(self):
                 self.model, self.optimizer, self.data_loader, num_epochs=2, cuda_device=[0, 1],
             )
 
+    @pytest.mark.skipif(not torch.cuda.is_available(), reason="No CUDA device registered.")
+    @pytest.mark.skipif(importlib.util.find_spec("apex") is None, reason="Apex is not installed.")
+    def test_trainer_can_run_amp(self):
+
+        self.model.cuda()
+        trainer = Trainer(
+            self.model,
+            self.optimizer,
+            self.data_loader,
+            num_epochs=2,
+            cuda_device=0,
+            opt_level="O1",
+        )
+        metrics = trainer.train()
+        assert "peak_cpu_memory_MB" in metrics
```

Comment:
You can delete these asserts, as they aren't related to this test.

Reply:
Done!